packages/desktop-client/src/components/modals/EditRule.js

```diff
@@ -789,7 +794,17 @@ export default function EditRule({
             <View style={{ flexShrink: 0 }}>
               <View style={{ marginBottom: 30 }}>
                 <Text style={{ color: colors.n4, marginBottom: 15 }}>
-                  If all these conditions match:
+                  If
+                  <FieldSelect
+                    style={{ display: 'inline-flex' }}
+                    fields={[
+                      ['every', 'all'],
```

Comment:
Any particular reason it’s `every` instead of `all` in the code?

Reply:
Was mapping it to the array methods (which are `every` / `some` instead of `all` / `any`). Happy to change to the latter pair if we prefer. At the end of the day those are just consts

Reply:
Changed to `and` / `or`